src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/helpers/TestUserHelper.kt

```diff
@@ -48,11 +51,18 @@ class TestUserHelper(private val password: String = TestUserHelper.defaultPasswo
             }
         }
         val token = tokenFetcher.getToken(email, password)
```

Comment:
I'm not completely sure why this is clearing roles, but adding in any that you need here seems to work...